controllers/netbot.js

```diff
@@ -2257,13 +2257,25 @@ class netBot extends ControllerBot {
     ])
 
     if (target != '0.0.0.0') {
-      await Promise.all([
+        /* todo
+          const stats = platform.getStats()
+          for(const stat  of stats){
+            stat => last24
+            stat => yesterday
+            stat => last3Days
+          }
+        */
+      const requiredPromises = [
         this.hostManager.yesterdayStatsForInit(jsonobj, target),
         this.hostManager.last60MinStatsForInit(jsonobj, target),
         this.hostManager.last30daysStatsForInit(jsonobj, target),
         this.hostManager.newLast24StatsForInit(jsonobj, target),
         this.hostManager.last12MonthsStatsForInit(jsonobj, target)
-      ])
+      ];
+      if (['gold', 'navy'].includes(platform.getName())) {
```

Comment:
1. use platform way to handle platform-specific features. do not include platform names here as if-else statement. let platform code to decide what specific interval of stats should be included.
2. better to remove hard code time range, use generic key `this.hostManager.stats` with a map like this
{ "30d": {flows...}}
3. keep all existing 24/30/60 unchanged